packages/image_picker/image_picker_android/android/src/main/java/io/flutter/plugins/imagepicker/ImageResizer.java

```diff
@@ -112,14 +133,28 @@ private void copyExif(String filePathOri, String filePathDest) {
     exifDataCopier.copyExif(filePathOri, filePathDest);
   }
 
-  private Bitmap decodeFile(String path) {
-    return BitmapFactory.decodeFile(path);
+  private Bitmap decodeFile(String path, @Nullable BitmapFactory.Options opts) {
+    return BitmapFactory.decodeFile(path, opts);
   }
 
   private Bitmap createScaledBitmap(Bitmap bmp, int width, int height, boolean filter) {
     return Bitmap.createScaledBitmap(bmp, width, height, filter);
   }
 
+  private int calculateInSampleSize(BitmapFactory.Options options, int reqWidth, int reqHeight) {
```

Comment:
The "in" in `inSampleSize` is identifying it as an input value (as opposed to, e.g., `outWidth` and `outHeight`, which are used for output); it doesn't make sense in the context of a return value.